pkg/volume/flexvolume/flexvolume.go

```diff
@@ -241,76 +245,142 @@ func (f flexVolumeMounter) GetAttributes() volume.Attributes {
 // flexVolumeManager is the abstract interface to flex volume ops.
 type flexVolumeManager interface {
 	// Attaches the disk to the kubelet's host machine.
-	attach(mounter *flexVolumeMounter) (string, error)
+	attach(mounter *flexVolumeMounter, volOptions map[string]string) (string, error)
 	// Detaches the disk from the kubelet's host machine.
 	detach(unmounter *flexVolumeUnmounter, dir string) error
```

Comment:
do you mind if I do that when I'll implement the detacher interface? I try to focus on attach here.